challenge_demo/README.md

```diff
@@ -0,0 +1,38 @@
+Responsible: Rokus
```

Comment:
Not yet correct. You can add a WIP-prefix to your PR as well when needed :smiley: